src/backend/api/reviews.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const { limit } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const allReviews = await knex('review').select('*');
+        res.json(allReviews);
+    } catch (error) {
+        throw error
+    }
+});
+
+router.post('/', async (req, res) => {
+    try {
+        const result = await knex('review').insert([{ title: req.body.title, meal_id: req.body.meal_id, created_date: req.body.created_date, description: req.body.description, stars : req.body.stars}]);
```

Comment:
```suggestion
        const newReview = { 
          title: req.body.title,
          meal_id: req.body.meal_id,
          created_date: req.body.created_date,
          description: req.body.description,
          stars : req.body.stars
        };
        
        // Validate the newReview before entering it into the database.
        
        const result = await knex('review').insert([newReview]);
```

Code becomes easier to read if it's not too long and does more things at once on the same line. Here you both create the review and enter it into the database at the same time. It also means you have no natural place to validate the input data.

Putting things into your database without checking it can be very, very dangerous in the real world.

Reply:
@miltonhultgren I appreciate your reviews alot, here I have a question, according to the mysql query, there are validations for the data entry, so shall I add more validations here as well?